wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json

```diff
@@ -0,0 +1,206 @@
+{
```

Comment:
The `$id` and `title` are IMHO optional and bring no value in this form, which seems autogenerated. I'd instead add `description` to the individual schema elements with more detailed comments on their behavior/intended use.

Reply:
Removed id and reduntant titles, added descriptions